package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "slf",
-  "version": "2.0.2",
+  "version": "2.0.3",
```

Comment:
`package-lock` must be updated as well. Always run `npm i` after updating the package version.